playbooks/openstack/openshift-cluster/launch.yml

```diff
@@ -55,7 +55,7 @@
     register: stack_show_status_result
```

Comment:
@lhuard1A thanks for the background info on the issue. Considering the limitations of the tools, I am good with the proposed changes.